### PR TITLE
PMM-5329 Update Prometheus to version 2.16.0

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/percona/pmm-server-packaging/pull/117

### CODE REVIEW

File: build/bin/build-server-rpm [hunk 1]
```diff
@@ -254,6 +254,7 @@ build() {
                 rpmbuild --define '_rpmdir %{_topdir}/RPMS/${spec_name}-${rpm_version}' --define 'dist .el7' -ba SOURCES/${spec_name}.spec
                 rm -f SOURCES/${spec_name}.spec*
             else
+                export PATH=/usr/local/go/bin/:$PATH
```
Comment (AlekSi) [inline build/bin/build-server-rpm]: @Sevlin Can we move that line and a few below to the spec file itself? That huge if/else code should become smaller over time, not larger.

Comment (Sevlin) [reply build/bin/build-server-rpm]: I think, the better way is to use alternatives, as we do for Grafana. Hence, we won't need to update `PATH` in runtime, build and dev envs.
```
sudo update-alternatives --install \"/usr/bin/go\" \"go\" \"/usr/local/go/bin/go\" 0
sudo update-alternatives --set go /usr/local/go/bin/go
```

Comment (adivinho) [reply build/bin/build-server-rpm]: $PATH variable hasn't changed after these commands

`sudo echo $PATH
/home/adivinho/golang/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin

sudo update-alternatives --install /usr/bin/go go /usr/local/go/bin/go 0
sudo update-alternatives --set go /usr/local/go/bin/go

sudo echo $PATH /home/adivinho/golang/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`

Comment (Sevlin) [reply build/bin/build-server-rpm]: @adivinho I've added alternative for `gofmt` in the Dockerfile

File: build/rpmbuild-docker/Dockerfile [hunk 2]
```diff
@@ -25,11 +25,13 @@ ENV GO_VERSION 1.13.7
 ENV GO_CHECKSUM b3dd4bd781a0271b33168e627f7f43886b4c5d1c794a4015abf34e99c6526ca3
 
 RUN wget --progress=dot:giga https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -O /tmp/golang.tar.gz && \
-    echo "${GO_CHECKSUM} /tmp/golang.tar.gz" | sha256sum -c -;
-RUN tar -C /usr/local -xzf /tmp/golang.tar.gz; \
+    echo "${GO_CHECKSUM} /tmp/golang.tar.gz" | sha256sum -c - && \
+    tar -C /usr/local -xzf /tmp/golang.tar.gz; \
```
Comment (AlekSi) [inline build/rpmbuild-docker/Dockerfile]: We use `--squash` do build this Docker image; there is no need to use `&&` pattern.
We should also not mix tabs and spaces.
We should also not mix things that should merged (that change) and things that should not (submodule branch update for FB) in one PR.

**Comment by adivinho** [comment]
@Sevlin Could u have a look at logs?
Looks like some dependencies are missed in order to build qan-app.

Screenshot_20200226_084022

**Comment by AlekSi** [comment]
@adivinho That's because that PR starts from the old `PMM-2.0` branch state. That issue was fixed already. Please create two new separate PRs after doing that:
```
git checkout PMM-2.0
git pull
git submodule update
```

**Pull request merged**